docs/reference/templating/index.md

```diff
@@ -0,0 +1,36 @@
+---
+description: In many places Bifröst can be customized using templating. How does they work?
+---
+
+# Templating
+
+In many places of the [configuration](../configuration.md) Bifröst can be not just customized using static strings or numbers, also templates can be used.
```

Comment:
In many places of the [configuration](../configuration.md) Bifröst can not only be customized using static strings or numbers, but templates can be used.